modules/ROOT/pages/cloud/overview/overview.adoc

```diff
@@ -0,0 +1,100 @@
+:page-layout: api-partial
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. For more information, see xref:ROOT:deploy:deployment-option/cloud/api/cloud-api-overview.adoc[Redpanda Cloud API Overview].
+
+== Control Plane API
+
+The Control Plane API enables you to programmatically manage your clusters, networks, and resource groups.
+
+The *base URL* of the Control Plane API for all organizations is:
+
+```
+https://api.redpanda.com
+```
+
+== Data Plane APIs
+
+The Data Plane APIs enable you to programmatically manage the resources within your clusters, including topics, users, access control lists (ACLs), and connectors.
+
+The *base URL* of the Data Plane APIs is unique to every cluster. You can retrieve this value by making a xref:get-/v1beta2/clusters/-id-[Get Cluster] request to your target cluster. Use the `dataplane_api.url` from the response body as the base URL when calling the Data Plane API endpoints. 
+
+== Quickstart
+
+. Authenticate to the API. See <<use-the-api-reference-to-interact-with-the-cloud-api>> if issuing requests on this page, or see the xref:ROOT:deploy:deployment-option/cloud/api/cloud-api-authentication.adoc[authentication guide] for details.
+. Create a resource group by making a xref:post-/v1beta2/resource-groups[`POST /v1beta2/resource-groups`] request.
+. Create a network by making a xref:post-/v1beta2/networks[`POST /v1beta2/networks`] request. When the long-running operation is complete, proceed to the next step.
+. Create a cluster by making a xref:post-/v1beta2/clusters[`POST /v1beta2/clusters`] request.
+. For BYOC, run `rpk cloud byoc`, passing the `metadata.cluster_id` from the Create Cluster response as a flag:
```

Comment:
It may be unclear what API (CP or DP) is supposed to be used here for these URLs. I assume this is gonna be a recurring problem, so wondering whether we should introduce something like a "CP API" and "DP API" pill which always clarifies whether we are referring to the API API or DP API.